modules/sfp_snov.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_snov
+# Purpose:      Spiderfoot plugin to search Snov.IO API for emails 
+#               associated to target domain
+#
+# Author:      Krishnasis Mandal <krishnasis@hotmail.com>
+#
+# Created:     16/05/2020
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+from netaddr import IPNetwork
+import urllib.request, urllib.parse, urllib.error
+import json
+
+class sfp_snov(SpiderFootPlugin):
+    """Snov:Investigate,Passive:Reputation Systems:client_id,client_secret:Gather available email ids from target domain"""
```

Comment:
This doesn't work the way you think it does :) Replace `client_id,client_secret` with `apikey`, as that's the flag that indicates this module requires credentials.

Reply:
Okay, got it. It is the one that is responsible for the API key required icon. 👍